.github/workflow_scripts/env_setup.sh

```diff
@@ -75,6 +75,7 @@ function install_all_no_tests {
 }
 
 function build_pkg {
+    pip install --upgrade setuptools wheel
```

Comment:
updated this package because test_install fails due to old version of setuptools